glue_plotly/html_exporters/scatter2d.py

```diff
@@ -16,45 +20,108 @@
 class PlotlyScatter2DStaticExport(Tool):
 
     icon = PLOTLY_LOGO
-    tool_id = 'save:plotly'
+    tool_id = 'save:plotly2d'
     action_text = 'Save Plotly HTML page'
     tool_tip = 'Save Plotly HTML page'
 
     def activate(self):
 
-        filename, _ = compat.getsavefilename(parent=self.viewer, basedir="plot.html")
-
-        fig = go.Figure(layout={'width': 800, 'height': 600})
+        filename, _ = compat.getsavefilename(
+            parent=self.viewer, basedir="plot.html")
+
+        width, height = np.array(self.viewer.figure.get_size_inches()*self.viewer.figure.dpi)[
+            0], np.array(self.viewer.figure.get_size_inches()*self.viewer.figure.dpi)[1]
+
+        # set the aspect ratio of the axes, the tick label size, the axis label sizes, and the axes limits
+        layout = go.Layout(
+            margin=dict(r=50, l=50, b=50, t=50),
+            width=1200,
+            height=1200*height/width,  # scale axis correctly
+            xaxis=dict(
+                title=self.viewer.axes.get_xlabel(),
+                titlefont=dict(
+                    family='Arial, sans-serif',
+                    size=self.viewer.axes.xaxis.get_label().get_size(),
+                    color='black'
+                ),
+                showticklabels=True,
+                tickfont=dict(
+                    family='Arial, sans-serif',
+                    size=self.viewer.axes.xaxis.get_ticklabels()[
+                        0].get_fontsize(),
+                    color='black'),
+                range=[self.viewer.state.x_min, self.viewer.state.x_max]),
+            yaxis=dict(
+                title=self.viewer.axes.get_xlabel(),
+                titlefont=dict(
+                    family='Arial, sans-serif',
+                    size=self.viewer.axes.yaxis.get_label().get_size(),
+                    color='black'),
+                range=[self.viewer.state.y_min, self.viewer.state.y_max],
+                showticklabels=True,
+                tickfont=dict(
+                    family='Old Standard TT, serif',
```

Comment:
Why is this a different font? Also maybe the font should be a variable/constant so it can be controlled from one place, e.g. ``DEFAULT_FONT = 'Arial, sans-serif'`` could be defined at the top of the file

Reply:
If you want to use a different font for ticks and tick labels, maybe you can create two constants. I personally wouldn't advocate mixing the two though, I think plots look a lot better with a consistent font.